release-notes.md

```diff
@@ -0,0 +1,65 @@
+# Release v0.8.0
+
+## New Features
+* Failed or Running Job can be cloned and resubmitted in Job detail page - [PR 1448](https://github.com/Microsoft/pai/pull/1448).
```

Comment:
> Failed or Running Job [](start = 1, length = 22)

Just "Jobs" should be enough, right? any job can be cloned.

Reply:
We have a problem to clone legacy jobs. @YitongFeng will fix it and all jobs will be supported to clone.